gradle/lib/dependabot/gradle/file_fetcher.rb

```diff
@@ -82,24 +87,28 @@ def subproject_buildfiles(root_dir)
           .subproject_paths
 
         subproject_paths.filter_map do |path|
+          @buildfile_name = T.let(@buildfile_name, T.nilable(String))
```

Comment:
This seems a little odd. Can you create a constructor and create it there?

Reply:
Please run it through **rubocop -A** to see if this autocorrects.  whitespace doesn't seem like a big deal, but that rule should just check for issues which have already been corrected and make the code less likely to have issues when someone else rebases.